MAUI/DataGrid/conditional-styling.md

```diff
@@ -382,4 +385,196 @@ public class ForeColorConverter : IValueConverter
+
+### Conditional styling of group summary cell using converter
+
+The appearance of group summary cell can be customized conditionally based on summary value by using `converter`, where converter returns the value based on summary value.
+
+{% tabs %}
+{% highlight xaml %}
+    <ContentPage.Resources>
+        <local:ColorConverter x:Key="converter" />
+        <Style TargetType="syncfusion:DataGridGroupSummaryCell">
+        <Setter Property="Background"
+                    Value="{Binding Converter={StaticResource converter}}" />
+        </Style>
+    </ContentPage.Resources>
+    <syncfusion:SfDataGrid x:Name="dataGrid" 
+                        ShowGroupDropArea="True"
+                        ItemsSource="{Binding OrderInfoCollection}">
+            <syncfusion:SfDataGrid.GroupSummaryRows>
+            <syncfusion:DataGridSummaryRow Title="salary {Salary}"
+                                            ShowSummaryInRow="True">
+                <syncfusion:DataGridSummaryRow.SummaryColumns>
+                    <syncfusion:DataGridSummaryColumn Name="Salary"
+                                                    MappingName="Salary"
+                                                    Format="{}{Sum:C0}"
+                                                    SummaryType="DoubleAggregate">
+                    </syncfusion:DataGridSummaryColumn>
+                </syncfusion:DataGridSummaryRow.SummaryColumns>
+            </syncfusion:DataGridSummaryRow>
+        </syncfusion:SfDataGrid.GroupSummaryRows>
+    </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+public class ColorConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var summaryValue = (value as SummaryRecordEntry).SummaryValues[0];
+        var aggregateValue = summaryValue.AggregateValues.ElementAt(0);
+        var calculatedValue = aggregateValue.Value;
+
+        //custom condition is checked.
+
+        if ((double)calculatedValue % 2 == 0)
+            return Colors.LightGreen;
+        return Colors.LightBlue;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Here, group summary cells are customized based on `Salary` summary value.
+
+![Conditional Style of DataGrid Group Summary Cell using Converter](maui-datagrid-conditional-groupsummary.png)
```

Comment:
Conditional styling of group summary cell using converter.

Reply:
modified